supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -69,6 +69,9 @@ object Run {
       rpargs.setOutputs(jesParameters.collect({ case i: JesFileOutput => i.name -> i.toGoogleRunParameter }).toMap.asJava)
       logger.debug(s"Outputs:\n${stringifyMap(rpargs.getOutputs.asScala.toMap)}")
 
+      val labels = Map("cromwell-workflow-id" -> s"cromwell-${workflow.id.toString}")
```

Comment:
FYI-- this is the **_subworkflow_**, not the outer workflow. The value `workflow` is defined as:

```scala
lazy val workflow = jobDescriptor.workflowDescriptor
```

I'm still looking to see if `Run` has a pointer to the main workflow's descriptor somewhere. I remember seeing something call breadcrumb, somewhere...

Also, don't call `s"cromwell-${workflow.id.toString}"`, just `s"cromwell-${workflow.id}"`.